src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx

```diff
@@ -54,6 +54,23 @@ To install the Fluent Bit plugin:
   If you'd rather not compile the plugin yourself, you can download pre-compiled versions from our [GitHub repository's releases page](https://github.com/newrelic/newrelic-fluent-bit-output/releases).
 </Callout>
 
+## Upgrade the Fluent Bit plugin [#upgrade-plugin]
+
+Before you upgrade your Fluent Bit plugin, run the following NRQL query to find the current versions of the output plugin being used in your system:
+
+```sql
+    FROM
+    K8sContainerSample select latest(containerImage) where podName like '%newrelic-logging%' facet clusterName
```

Comment:
@paperclypse I'd recommend we use the formatting in our style guide for queries:

FROM  K8sContainerSample 
SELECT latest(containerImage) 
WHERE podName like '%newrelic-logging%' 
FACET clusterName

Reply:
Good call